components/help/help_controller.tsx

```diff
@@ -13,12 +11,17 @@ import Formatting from './components/formatting';
 import Attaching from './components/attaching';
 import Commands from './components/commands';
 
-export default class HelpController extends React.Component {
-    componentDidUpdate() {
-        ReactDOM.findDOMNode(this).scrollIntoView();
-    }
+type HelpControllerProps = {
+    match: {
+        url: string;
+    };
+};
 
-    render() {
+export default class HelpController extends React.Component<HelpControllerProps, {}> {
+    public componentDidUpdate() {
+        (ReactDOM.findDOMNode(this) as HTMLInputElement).scrollIntoView();
```

Comment:
I think this type might be incorrect. I'm pretty sure this will resolve to an `HTMLDivElement`. Also, would you mind switching this to use the parameterized version of `findDOMNode` instead of using `as`?

```suggestion
        ReactDOM.findDOMNode<HTMLDivElement>(this).scrollIntoView();
```